README.en-US.md

```
@@ -0,0 +1,132 @@
+
+
+# Demo Preview
+<div align="center">
+  <img src="https://github.com/zhou19830318/doubao_ai_agent_esp32/blob/main/ezgif-257beaf8d11884.gif">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)